test/functionalTest/cases/0876_entity_dates/filter_datecreated_sub.test

```diff
@@ -26,7 +26,7 @@ Filter with dateCreated in subscriptions
 --SHELL-INIT--
 dbInit CB
 brokerStart CB
-accumulatorStart
+accumulatorStart --pretty-print
```

Comment:
I would propose the following:

New issue about fixing brokerStart (you know I've been complaining about that function quite some time), where we 'migrate' brokerStart and probably a few more function to behave like orionCurl.  accumulator.py included.